activesupport/lib/active_support/notifications.rb

```diff
@@ -244,6 +244,35 @@ def subscribe(pattern = nil, callback = nil, &block)
         notifier.subscribe(pattern, callback, monotonic: false, &block)
       end
 
+      # Subscribe to an event name with the passed +block+.
+      #
+      # To subscribe to events, pass a String to match exact event
+      # names, or pass a Regexp to match all events that match a pattern.
+      #
+      # The +block+ will receive five parameters with information about the event:
+      #
+      #   ActiveSupport::Notifications.monotonic_subscribe('render') do |name, start, finish, id, payload|
+      #     name    # => String, name of the event (such as 'render' from above)
+      #     start   # => Monotonic time, when the instrumented block started execution
+      #     finish  # => Monotonic time, when the instrumented block ended execution
+      #     id      # => String, unique ID for the instrumenter that fired the event
+      #     payload # => Hash, the payload
+      #   end
+      #
+      # The +start+ and +finish+ values above represent monotonic time.
+      #
+      # If the block passed to the method only takes one parameter,
+      # it will yield an event object to the block:
+      #
+      #   ActiveSupport::Notifications.monotonic_subscribe(/render/) do |event|
+      #     @event = event
+      #   end
+      #
+      # Raises an error if invalid event name type is passed:
+      #
+      #  ActiveSupport::Notifications.monotonic_subscribe(:render) {|*args| ...}
+      #  #=> ArgumentError (pattern must be specified as a String, Regexp or empty)
+      #
```

Comment:
I see that most of this is copied [from `ActiveSupport::Notifications.subscribe`](https://github.com/rails/rails/blob/4dafe6a1eb38b834eea98f2c8e9e808b0579b58a/activesupport/lib/active_support/notifications.rb#L212-L243).  Instead of doing that, let's just explain `monotonic_subscribe` in terms of `subscribe`, and explain when the user might want to use `monotonic_subscribe` instead of `subscribe`.

Reply:
Your feedback is greatly appreciated!  I'll make the changes.